.github/workflows/ci-backend.yml

```diff
@@ -120,7 +126,6 @@ jobs:
               run: |
                   sudo apt-get update
                   sudo apt-get install libxml2-dev libxmlsec1-dev libxmlsec1-openssl
-              if: steps.python.outputs.cache-hit != 'true'
```

Comment:
This in particular was a problem, since"Install python dependencies" fails without this step